corona-warn-app/cwa-app-android
# Refactor to camerax / boofCV, POC (EXPOSUREAPP-11089)

This PR addresses EXPOSUREAPP-11089.

**Contains:** 
Refactored UQS Scanner-Fragment and related classes.
Refactored Submission On-behalf Scanner Fragment and related classes.
Use of newest version of CameraX/boofCV.

**Note:**
POC for testing!

**How to test:**
- Open the Universal QR Code Scanner inside the app
- Scan some certificates
- Check if everything works fine
